backend/pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+DJANGO_SETTINGS_MODULE = ickly.test_settings
+addopts = --reuse-db
```

Comment:
These tests take forever initially because it is running the data migration to create the test database. I added the `--reuse-db` option so that it doesn't flush the test db it created each time the tests are run.